subprojects/core-api/core-api.gradle

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+apply plugin: 'java-library'
+
+sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+
+dependencies {
+    api project(":baseServices")
+    api project(":baseServicesGroovy")
+    api project(":buildCache")
+    api project(":logging")
+    api project(":modelCore")
+    api project(":persistentCache")
+    api project(":processServices")
+    api project(":resources")
+
+    implementation libraries.ant
+    implementation libraries.commons_io
+    implementation libraries.commons_lang
+    implementation libraries.jcip
+
+    testFixturesImplementation project(":internalTesting")
+}
+
+useTestFixtures(project: ":logging")
+//useClassycle()
```

Comment:
This is what we do elsewhere, too, to signal that this should eventually be enabled.